src/store/index.js

```diff
@@ -0,0 +1,81 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import * as utils from './utils';
+
+Vue.use(Vuex);
+
+
+export default new Vuex.Store({
+  state: {
+    firstStart: true,
```

Comment:
Can't find usage of this variable. Do we need it?

Reply:
It seems like not